npm-debug.log

```diff
@@ -0,0 +1,19 @@
+0 info it worked if it ends with ok
```

Comment:
This file should be omitted from your repository (add `*.log` into your `.gitignore` file).